pkg/ddc/vineyard/operations/base.go

```diff
@@ -39,21 +41,42 @@ func NewVineyardFileUtils(podNamePrefix string, port int32, replicas int32, name
 	}
 }
 
+func validateResourceName(name string) error {
+	validNameRegex := `^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`
```

Comment:
You can check if  validation.IsDNS1035Label can satisfy your requirement. And I think you can also check https://github.com/asaskevich/govalidator/blob/a9d515a09cc289c60d55064edec5ef189859f172/validator.go#L674

Reply:
Done.